### DISCUSSION TITLE
Mi air purifier elite support

### DISCUSSION THREAD

**Discussion by asim1701** [rytilahti/python-miio#1930] [General]
Hello

can you add support for zhimi.airp.meb1?

https://www.mi.com/global/product/xiaomi-smart-air-purifier-elite/

**Comment by rytilahti** [comment]
This is likely already supported by the `genericmiot`, i.e., `miiocli genericmiot --ip <addr> --token <token> status` should work with the current master branch or dev0 pre-release (`pip install python-miio==0.6.0.dev0`).
